Xamarin.Forms.Platform.iOS/VisualElementPackager.cs

```diff
@@ -54,6 +54,22 @@ protected virtual void Dispose(bool disposing)
 
 			if (disposing)
 			{
+				if (ElementController != null)
+				{
+					for (var z = 0; z < ElementController.LogicalChildren.Count; z++)
```

Comment:
Any reason why you used `z` here instead of a typical `i`?

Reply:
Because I just copy and pasted the structure from lower down in the code where it iterates over the children :-)  Not a great reason but it's a reason